.eslintrc.json

```diff
@@ -19,7 +19,12 @@
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
     "no-shadow": "off",
-    "arrow-parens": ["error", "as-needed"]
+    "arrow-parens": ["error", "as-needed"],
+    "no-unused-vars": "off",
+    "react/prefer-stateless-function": "off",
+    "no-console": "off",
+    "react/prop-types": "off",
+    "class-methods-use-this": "off"
```

Comment:
Opps, we have some little changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json)

![image](https://user-images.githubusercontent.com/31897434/73417568-ca44bd00-4318-11ea-8a5e-fedabd629b8c.png)
[Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)

Reply:
I did mention that in the PR summary, I'll change it ok! Thank you.

Reply:
And you've mentioned that I made 'almost' all the changes, what did I leave?

Reply:
Just the linter's config file.